### ISSUE TITLE
Luxcore dosn't work on blender 3.6

### ISSUE THREAD

**Issue by SlincHD** [LuxCoreRender/BlendLuxCore#832]
![image](https://github.com/LuxCoreRender/BlendLuxCore/assets/85231739/6d4d33e8-6cb0-4202-9ad4-512afeb329c7)
i get this error code when trying to render on Luxcore with blender 3.6

**Comment by Beyond1nfinity** [comment]
![blender_73Pze5f7vC](https://github.com/LuxCoreRender/BlendLuxCore/assets/84192277/4da59356-040e-4a3b-bc4a-15163562f0ef)
yeah i am getting the same issue as well

**Comment by BUGProtectionAssociation** [comment]
me too
![image](https://github.com/LuxCoreRender/BlendLuxCore/assets/65578116/02c7b8ea-b7cd-4835-a918-7f7845d3219b)

**Comment by papataci** [comment]
Same error here, Blender 3.6 Linux Mint 21.1

**Comment by foreachthing** [comment]
✋🏼 
Maybe this might help:
```
ERROR: list index out of range
Traceback (most recent call last):
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\BlendLuxCore\engine\viewport.py", line 76, in view_update
    engine.session = engine.exporter.create_session(depsgraph, context, engine=engine)
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\BlendLuxCore\export\__init__.py", line 110, in create_session
    instances = self.object_cache2.first_run(self, depsgraph, view_layer, engine, luxcore_scene,
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\BlendLuxCore\export\caches\object_cache.py", line 295, in first_run
    self._convert_obj(exporter, dg_obj_instance, obj, depsgraph, luxcore_scene,
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\BlendLuxCore\export\caches\object_cache.py", line 364, in _convert_obj
    exported_stuff = self._convert_mesh_obj(exporter, dg_obj_instance, obj, obj_key, depsgraph,
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\BlendLuxCore\export\caches\object_cache.py", line 435, in _convert_mesh_obj
    exported_mesh = mesh_converter.convert(obj, mesh_key, depsgraph, luxcore_scene,
  File "C:\Users\USER\AppData\Roaming\Blender Foundation\Blender\3.6\scripts\addons\BlendLuxCore\export\mesh_converter.py", line 104, in convert
    mesh_definitions = luxcore_scene.DefineBlenderMesh(mesh_key, loopTriCount, loopTriPtr, loopPtr,
IndexError: list index out of range
```

**Comment by neo2068** [comment]
This is a known error. There is a separate branch for_blender3.6 for the work to make LuxCore compatible with Blender 3.6. There are many changes in the Blender API which breaks backward compability. It will be the same for Blender 4.0.

**Comment by HowardBlandy** [comment]
Still can't work after installed the latest update, here's the error log from BlendLuxCore

Python argument types in
    Scene.DefineBlenderMesh(Scene, str, int, int, int, int, int, int, int, bool, list, list, int, int, NoneType, tuple, list, NoneType)
did not match C++ signature:
    DefineBlenderMesh(class luxcore::detail::SceneImpl * __ptr64, class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, class boost::python::api::object, class boost::python::api::object, unsigned __int64, unsigned int, class boost::python::tuple, class boost::python::api::object, class boost::python::api::object)
    DefineBlenderMesh(class luxcore::detail::SceneImpl * __ptr64, class std::basic_string<char,struct std::char_traits<char>,class std::allocator<char> >, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, unsigned __int64, class boost::python::api::object, class boost::python::api::object, unsigned __int64, unsigned int, class boost::python::api::object, class boost::python::tuple, class boost::python::api::object, class boost::python::api::object)

**Comment by neo2068** [comment]
The update was not finished. Now, the the latest version can be found here [Link](https://github.com/LuxCoreRender/BlendLuxCore/releases/tag/blendluxcore_v2.7beta1). Please retest again.

**Comment by BUGProtectionAssociation** [comment]
> The update was not finished. Now, the the latest version can be found here [Link](https://github.com/LuxCoreRender/BlendLuxCore/releases/tag/blendluxcore_v2.7beta1). Please retest again.

Thanks. It worked

**Comment by SlincHD** [comment]
> The update was not finished. Now, the the latest version can be found here [Link](https://github.com/LuxCoreRender/BlendLuxCore/releases/tag/blendluxcore_v2.7beta1). Please retest again.

yes i can confirm that it work on blender 3.6

**Comment by SlincHD** [comment]
i also cold confirm that it works on windows 10 and 11 but 11 had some complication that i could fix that where related to #831
